collections/St_Johns_College/SW_V1_St_Johns_College_MS_062.xml

```diff
@@ -0,0 +1,213 @@
+                           </locusGrp>
+                        </condition>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="42">In double columns, each column
+                              <dimensions type="column" unit="mm">
+                              <height min="213" max="215">213–15</height>
+                              <width quantity="62">62</width>
+                           </dimensions>, with <dimensions type="intercolumn" unit="mm">
+                              <width quantity="15">15</width>
+                           </dimensions>between columns, in 42 lines to the column. No prickings;
+                           bounded and ruled in black ink.</layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote script="protogothic">Written in very late <term type="script">protogothic bookhand</term>,
+                        square and regular, above top line; <locus from="1" to="2">fols.
+                           1–2</locus> are by a different contemporary hand. Punctuation by <term
+                           type="punctuation">point</term>, <term type="punctuation">punctus
+                           elevatus</term>, and <term type="punctuation">punctus
+                           interrogativus</term>.</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="rubrication">Headings in red.</decoNote>
+                     <decoNote type="flourInit">The only other decoration is 3-line red, blue, and
+                        green arabesque capitals at the heads of the sermons, with infill in a
+                        contrasting colour.</decoNote>
+                     <decoNote type="flourInit">In item 1, headings in red, alternate 1-line red and
+                        green arabesque capitals to divide the verses.</decoNote>
+                  </decoDesc>
+                  <bindingDesc>
+                     <p>A modern replacement. Sewn on five thongs. At the front, three modern paper
+                        flyleaves and one of medieval vellum, perhaps earlier a pastedown; at the
+                        rear, three modern paper flyleaves (v–vii).</p>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notBefore="1190" notAfter="1225">s. xii/xiii or
+                        s. xiii in.</origDate>
+                     <origPlace>
+                        <country key="place_7002445">England</country>
+                     </origPlace>
+                  </origin>
+                  <provenance>‘Hic liber est ecclesie sancte Marje de suwicha quem ab ea alienari
+                     non licet Quod qui facere presumpserit siue titulum istum dolose deleuerit uel
```

Comment:
the name in l. 164 needs to be marked up with `orgName role="fmo"` (the reference is to Southwick, hants, Augustinian priory, which we don't have an authority record for yet)